src/commands/add-rm.ts

```diff
@@ -8,7 +8,8 @@ export async function addCommand(addValue: string, commandOptions: CommandOption
   const {cwd, config, pkgManifest} = commandOptions;
   let [pkgName, pkgSemver] = addValue.split('@');
   if (!pkgSemver) {
-    const body = (await got(`http://registry.npmjs.org/${pkgName}/latest`).json()) as any;
+    const getJson = bent('https://registry.npmjs.org', 'json', 200);
```

Comment:
@drwpow @FredKSchott I forgot to mention in the changes that I swapped this to use `https:` instead of `http:`. Was there a reason we were using the insecure version? It didn't auto-redirect to a secure version.

Reply:
I think @FredKSchott would know more but I can’t think of a reason off the top of my head why HTTP would be better